serialization/InformationModel/README.md

```diff
@@ -0,0 +1,307 @@
+# SPDX v3 Information Based Serialization
+
+The SPDX logical model defines meaning, relationships and shapes of the SPDX element graph, but the same
+information can be serialized into many different byte sequences.  A physical data model (concrete schema)
+specifies logical element serialization using a specific data format, while an information model
+(abstract schema) specifies logical element serialization using any data format.  Reading information
+into applications from multiple formats allows instances to be compared; writing it to multiple formats
+allows it to be converted between formats without loss.
+
+### Serialization Concepts
+
+Information exists in the minds of users (producers and consumers), in the state of applications running
+on systems, and in the data exchanged among applications.
+Serialization converts application information into byte sequences (a.k.a. protocol data units, messages,
+payloads, information exchage packages) that can be validated, communicated and stored.
+De-serialization parses payloads back into application state.
+Serialization is not a goal in and of itself, it is the mechanism by which applications exchange information
+in order to make it available to users.
+
+![Information](images/interoperability.jpg)
+
+Serialization should be:
+1) **lossless**, so that information is not modified in transit and all applications have the identical information
+2) **transparent**, so that information is unaffected by whether or how it has been serialized;
+users should not know or care.
+
+Shannon's information theory defines the relationship between information and serialization (coding).
+Mathematicians characterize conditions applied to a mechanism as *necessary* and/or *sufficient*:
+a serialization that omits necessary data loses information, one that uses more data than sufficient
+conveys no extra information.
+TV bartender/philosopher Ted Danson put it most succinctly: a necessary and sufficient serialization
+mechanism has "everything you need and nothing you don't."
+
+### SPDX v3 Information
+
+*NOTE: The examples in this document are based on the rapidly-evolving SPDX v3 logical model.
+As decisions are made and the logical model stabilizes, the examples will be updated to conform.*
+
+The classes of a logical model are either referenceable or datatypes.
+* A **datatype** class is known only by its value.
+Two instances of the class are equivalent if and only if every property in the class is equal.
+* A **referenceable** class has a unique identifier (primary key) and each instance is known by its id.
+Two instances of the class are considered equivalent if and only if their ids are equal.
+If only a single value can correspond to an id, the class is a *mapping* and colliding values
+are treated as an error.
+
+SPDX v3 organizes all of its information into a single referenceable "Element" class - all logical
```

Comment:
Just a note for future reference - there is an[ open question](https://github.com/spdx/spdx-3-model/issues/224#issuecomment-1527294548) as to whether a License should be considered a peer of "Element" or a subclass of element.  The outcome of this may impact the above statement.

Reply:
Yes.  An information model can define an arbitrary number of referenceable class types, with or without inheriting from one class to another.  It looks like they are leaning toward making license entries Elements, but I have some reservations about throwing so much stuff (like extensions) into the base class.  I might define Element without the kitchen sink, and ExtensableElement from it, and if there is no plausible use case for adding extensions to Licenses and Annotations, they'd inherit from the simpler base.

Thanks for the heads up.